client/src/main.rs

```diff
@@ -1257,7 +1304,11 @@ async fn perf_workload(
     let big_end = big_start.elapsed();
 
     guest.flush(None)?;
-    perf_summary("rwrites", count, io_depth, wtime, big_end);
+    perf_summary("rwrites", count, io_depth, wtime.clone(), big_end);
+    if wtr.is_some() {
```

Comment:
@steveklabnik, in addition to explaining to be why `wtr: &mut Option<&mut csv::Writer<File>>` works, pointed out I can `if let ..` this part which I will do.